docs/development/nodejs/how-to-install-nodejs-and-nginx-on-centos-8/index.md

```diff
@@ -0,0 +1,303 @@
+---
+author:
+    name: Linode
+    email: docs@linode.com
+description: 'In this guide you will install and configure NGINX to serve static site content on an CentOS 8 Linode. You will also create a Node.js server and use NGINX as a reverse proxy to your Node.js server. To test your configurations, you will create an index.html file as your static content and a test JavaScript file to be served by your Node.js server. '
+og_description: 'In this guide you will install and configure NGINX to serve static site content on an CentOS8 Linode. You will also create a Node.js server and use NGINX as a reverse proxy to your Node.js server. To test your configurations, you will create an index.html file as your static content and a test JavaScript file to be served by your Node.js server.'
+keywords: ["linode guide", "hosting a website", "website", "linode setup", " install node.js", " install nginx", "centos", " front-end requests", " back-end requests"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-04-14
+modified_by:
+    name: Linode
+published: 2020-04-14
+title: 'How to Install Node.js and NGINX on CentOS 8'
+h1_title: 'Installing Node.js and NGINX on CentOS 8'
+external_resources:
+ - '[Node.js](http://nodejs.org)'
+ - '[NGINX](http://nginx.com/)'
+ - '[NodeSchool](http://nodeschool.io/)'
+ - '[Node Version Manager](https://github.com/creationix/nvm)'
+ - '[npm](https://www.npmjs.com/)'
+audiences: ["intermediate"]
+concentrations: ["Web Applications"]
+languages: ["javascript"]
+---
+
+Node.js is an open-source JavaScript runtime environment that can serve dynamic and responsive content and is often used to create and serve web applications. When serving Node.js applications, NGINX is commonly used to create a reverse proxy that points at a running Node.js server. In this guide, you will install and configure NGINX to handle requests to static files, like `index.html` and also, create a reverse proxy to a Node.js server. You will then create a test JavaScript file in order to test your running Node.js server.
```

Comment:
Can we bake the distro into the opening paragraph somewhere? I think this will help SEO. Need it for all the guides in this PR.

Reply:
Okay, I have not heard of this and haven't been doing it in guides I draft or tech edit. Let me know if this is something we should be doing moving forward, please.

Reply:
Yes this is something we should do moving forward.